Library/Homebrew/utils/github.rb

```diff
@@ -342,8 +342,8 @@ def merge_pull_request(repo, number:, sha:, merge_method:, commit_message: nil)
     open_api(url, data: data, request_method: :PUT, scopes: CREATE_ISSUE_FORK_OR_PR_SCOPES)
   end
 
-  def print_pull_requests_matching(query)
-    open_or_closed_prs = search_issues(query, type: "pr", user: "Homebrew")
+  def print_pull_requests_matching(query, only)
+    open_or_closed_prs = search_issues(query, is: only, type: "pr", user: "Homebrew")
```

Comment:
Is this handled as expected when `only` is `nil`?

Reply:
Yes,  the result when `only` is `nil` is below.

<details><summary>brew search ruby --pul-request</summary><div>

```
==> Open pull requests
licensee 9.14.1 (new formula) (https://github.com/Homebrew/homebrew-core/pull/67513)
colorls 1.4.3 (new formula) (https://github.com/Homebrew/homebrew-core/pull/67165)
tpp: fix build with Xcode 12 (https://github.com/Homebrew/homebrew-core/pull/66517)
vowpal-wabbit 8.9.0 (https://github.com/Homebrew/homebrew-core/pull/64736)
dashing: update (https://github.com/Homebrew/homebrew-core/pull/65648)
install.sh: preload homebrew/core before brew update (https://github.com/Homebrew/install/pull/371)
tomee-plume 8.0.5 (https://github.com/Homebrew/homebrew-core/pull/65750)
Adds support for ManyCam (https://github.com/Homebrew/homebrew-cask/pull/91231)
minidlna 1.3.0 (https://github.com/Homebrew/homebrew-core/pull/65760)
Fix/m4/patch (https://github.com/Homebrew/linuxbrew-core/pull/21848)
mermaid-cli 8.8.4 (new formula) (https://github.com/Homebrew/homebrew-core/pull/65521)
openjdk: Add support for Apple silicon (https://github.com/Homebrew/homebrew-core/pull/65670)
alluxio 2.4.1 (https://github.com/Homebrew/homebrew-core/pull/65422)
autoconf 2.70 (https://github.com/Homebrew/homebrew-core/pull/66511)
geant4 10.7.0 (https://github.com/Homebrew/homebrew-core/pull/66422)
workflows: add remove-disabled-formulae workflow (https://github.com/Homebrew/homebrew-core/pull/67386)
Check flavour to prevent mixed architectures (https://github.com/Homebrew/brew/pull/9126)
go update gobootstrap on arm mac to a native arm version (https://github.com/Homebrew/homebrew-core/pull/67154)
onedrive-linux-client 2.4.5 (new formula) (https://github.com/Homebrew/homebrew-core/pull/64256)
ldc: use llvm (https://github.com/Homebrew/homebrew-core/pull/61117)
build(deps): bump rubocop from 1.5.1 to 1.6.1 in /Library/Homebrew (https://github.com/Homebrew/brew/pull/10050)

==> Closed pull requests
ruby-build 20201221 (https://github.com/Homebrew/homebrew-core/pull/67338)
utils/ruby: silence `which` errors when finding ruby (https://github.com/Homebrew/brew/pull/10034)
ruby-install 0.8.1 (https://github.com/Homebrew/homebrew-core/pull/67413)
utils/ruby.sh: simplify and fix Ruby-related logic (https://github.com/Homebrew/brew/pull/9472)
utils/ruby.sh: various tweaks (https://github.com/Homebrew/brew/pull/9544)
ruby-build 20201210 (https://github.com/Homebrew/homebrew-core/pull/66624)
ruby@2.4: deprecate! (https://github.com/Homebrew/homebrew-core/pull/66473)
ruby-build 20201208 (https://github.com/Homebrew/homebrew-core/pull/66470)
ruby-install 0.8.0 (https://github.com/Homebrew/homebrew-core/pull/66784)
cleanup: fix portable Ruby behaviour. (https://github.com/Homebrew/brew/pull/9479)
Fix macOS Ruby version handling (https://github.com/Homebrew/brew/pull/9465)
libprelude: explicitly disable Python 2 bindings and depend on ruby@2.6 for ruby bindings (https://github.com/Homebrew/linuxbrew-core/pull/21756)
ruby-build 20201118 (https://github.com/Homebrew/homebrew-core/pull/65111)
ruby-build 20201117 (https://github.com/Homebrew/homebrew-core/pull/65026)
Dependabot/bundler/library/homebrew/ruby macho 2.5.0 (https://github.com/Homebrew/brew/pull/10104)
tests.yml: force vendored Ruby on Catalina (https://github.com/Homebrew/homebrew-core/pull/66290)
Merge 2020-12-09 4f5a1e9aa59 (https://github.com/Homebrew/linuxbrew-core/pull/21752)
brew.sh: don't allow system Ruby on Catalina. (https://github.com/Homebrew/brew/pull/9442)
Improved no_usable_ruby() test (https://github.com/Homebrew/install/pull/338)
fq 0.13.4 (https://github.com/Homebrew/homebrew-core/pull/66917)
...
```

</div></details>